numpy/core/fromnumeric.py

```diff
@@ -2086,15 +2086,25 @@ def clip(a, a_min, a_max, out=None, **kwargs):
     --------
     :ref:`ufuncs-output-type`
 
+    Notes
+    --------
+    When `a_min` is greater than `a_max`, `clip` returns an 
+    array in which all values are equal to `a_max`, 
+    as shown in the second example.  
+
     Examples
     --------
     >>> a = np.arange(10)
-    >>> np.clip(a, 1, 8)
-    array([1, 1, 2, 3, 4, 5, 6, 7, 8, 8])
     >>> a
     array([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+    >>> np.clip(a, 1, 8)
+    array([1, 1, 2, 3, 4, 5, 6, 7, 8, 8])
+    >>> np.clip(a, a_max = 1, a_min = 8)
```

Comment:
```suggestion
    >>> np.clip(a, 8, 1)
```

Reply:
I have included your two suggestions. But the circle-ci checks fail, how can I resolve this?

Reply:
You can't, there is something just broke with the release notes creation (maybe `towncrier` master has an incompatible change, I am not sure yet).  The docs build fine though, so this could be merged as is right now.

Reply:
As a follow up question should there be a regression test for this behavior of `clip()`?

Like:

```   
def test_clip_special_case(self):
        arr = [-1, 5, 2, 3, 10, -4, -9]
        out = np.clip(arr, 8, 1)
        tgt = [1, 1, 1, 1, 1, 1, 1]
        assert_equal(out, tgt)
```